deploy-to-aks/action.yml

```diff
@@ -0,0 +1,150 @@
+name: Deploy to AKS environment
+description: Deploy a docker image to an AKS environment
+
+inputs:
+  azure-credentials:
+    description: A JSON string containing service principle credentials.
+    required: true
+  gcp-wip:
+    description: The full identifier of the GCP Workload Identity Provider
+    required: false
+  environment:
+    description: Environment to deploy to
+    required: true
+  github-token:
+    description: Default Github token retrieved via secrets.GITHUB_TOKEN or PAT with permission to the repository (Required)
+    required: true
+  pr-number:
+    description: PR number for the review app
+    required: false
+  sha:
+    description: Commit sha to be deployed
+    required: true
+  slack-webhook:
+    description: Name of the slack webhook
+    required: false
+  db-seed:
+    description: Run seed command after a deployment
```

Comment:
Maybe repeat it's normally only for review apps

Reply:
added note